app/routes/context_window.py

```
@@ -0,0 +1,940 @@
+
+    Authentication:
+    - Pass token as query parameter: ws://host/api/context-window/{id}/ws?token=xxx
+    - Token should be a valid JWT or session token
+    """
+    # Validate authentication token
+    user_id = None
+    if token:
+        try:
+            from app.middleware.auth import validate_token
+            user_id = await validate_token(token)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)